src/Microsoft.TestPlatform.Extensions.BlameDataCollector/BlameCollector.cs

```diff
@@ -499,6 +503,7 @@ private void EventsTestCaseEnd(object? sender, TestCaseEndEventArgs e)
     /// <param name="args">SessionEndEventArgs</param>
     private void SessionEndedHandler(object? sender, SessionEndEventArgs args)
     {
+        _testSessionEndedTimestamp = DateTime.Now;
```

Comment:
There is no behavior associated with the time here, and all the methods that take it a decide take it as a parameter, testing the behavior would require no special infrastructure. And we are testing it, because we have all approaches tested (crash dump, collect always crash dump, and collect failfast.

Comparing Now or UtcNow, neither is riskier than the other, in both cases we get the timestamp locally, and get both in the same ways, so it does not matter much if it is offset by timezone or not. But let me change that to UtcNow.

Reply:
It can't be fully reliable anyway because we are racing between procdump seeing that process is exiting (when a breakpoint that it sets is reached), and handling the event that testhost sent to us in eventhandler.

Reply:
Ok fine by me - keeping open for the `UtcNow` change if you want to do it, otherwise feel free to close.

Reply:
The unreliability shown in our CI, I think I will have to roll back most of this, and just rely on the test count as it was before to see if we crashed, and only use the output for waiting for procdump, but not for crash detection.

Reply:
reverted that, now I just use the output for detection if we are currently crashdumping, but not for detection if any crash dump was made.